IBPSA/Utilities/IO/SignalExchange/Read.mo

```diff
@@ -7,6 +7,9 @@ model Read "Block that allows a signal to be read as an FMU output"
   parameter SignalTypes.SignalsForKPIs KPIs = SignalTypes.SignalsForKPIs.None
     "Tag with the type of signal for the calculation of the KPIs";
 
+  parameter String zone = "1" if (Integer(KPIs)==2 or Integer(KPIs)==3 or Integer(KPIs)==4 or Integer(KPIs)==5 or Integer(KPIs)==6) "Zone designation, required if KPIs is AirZoneTemperature, RadiativeZoneTemperature, OperativeZoneTemperature, RelativeHumidity, or CO2Concentration"
+  annotation(Dialog(enable=(Integer(KPIs)==2 or Integer(KPIs)==3 or Integer(KPIs)==4 or Integer(KPIs)==5 or Integer(KPIs)==6)));
+
```

Comment:
@dhblum Please break these two very long lines, then it is ready for review by a non-LBL person.

Reply:
Addressed in https://github.com/ibpsa/modelica-ibpsa/pull/1295/commits/499a1f458e5e0149089584bb5c9d4f4d80cac23a.

Reply:
@dhblum can't you use `KPI==SignalTypes.SignalsForKPIs. AirZoneTemperature or KPI==...` instead of using an integer conversion? This way, if you later add a new KPI, or reorder them, the integer indices don't have to be adjusted and the code is better readable.

Reply:
Same comment for the Dialog annotation.

Reply:
Yes that works too, and I prefer as well.  Thanks!

Addressed in https://github.com/ibpsa/modelica-ibpsa/pull/1295/commits/8302e10ff07f1c3cedd10190f951606134372451.